hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/dstable/DistributionTable.java

```diff
@@ -310,15 +304,17 @@ protected Object getItemIdToSelect() {
     }
 
     @Override
-    protected void publishEntityAfterValueChange(final DistributionSet selectedLastEntity) {
+    protected void publishSelectedEntityEvent(final DistributionSet selectedLastEntity) {
         eventBus.publish(this, new DistributionTableEvent(BaseEntityEventType.SELECTED_ENTITY, selectedLastEntity));
-        if (selectedLastEntity != null) {
-            managementUIState.setLastSelectedEntity(selectedLastEntity.getId());
+        if (selectedLastEntity == null) {
```

Comment:
The behavior of this method is almost the same of every subclass. As far as I understood was identical selection handling one of the triggers for this pull request. May I ask you to (at least) try to generalize this - I know it was so before already.